junebug/channel.py

```diff
@@ -259,6 +259,7 @@ def _application_config(self):
             'redis_manager': self.config.redis,
             'inbound_ttl': self.config.inbound_message_ttl,
             'outbound_ttl': self.config.outbound_message_ttl,
+            'message_rate_bucket': self.config.inbound_message_rate_bucket,
```

Comment:
Should the config variable really be `inbound_message_rate_bucket`? I'm a bit confuse by the `inbound_` prefix on the one name and not on the other. Also wondering about the name of this option -- `message_rate_bucket` seems to imply the name of a bucket, rather than the size of a bucket.

Wondering what a better name is. Maybe `message_rate_window`? Or `metric_window`?